plasma_framework/test/src/framework/registries/ExitGameRegistry.test.js

```diff
@@ -0,0 +1,69 @@
+const ExitGameRegistry = artifacts.require('ExitGameRegistry');
+
+const { BN, expectRevert } = require('openzeppelin-test-helpers');
+const { expect } = require('chai');
+
+contract('ExitGameRegistry', ([operator, other, dummyAccount]) => {
+    beforeEach(async () => {
+        this.registry = await ExitGameRegistry.new();
+        this.dummyExitGameAddress = (await ExitGameRegistry.new()).address;
+    });
+
+    describe('exitGames', () => {
+        beforeEach(async () => {
+            this.dummyTxType = 1;
+            await this.registry.registerExitGame(this.dummyTxType, this.dummyExitGameAddress);
+        });
+
+        it('can receive exit game address with tx type', async () => {
+            expect(await this.registry.exitGames(this.dummyTxType)).to.equal(this.dummyExitGameAddress);
+        });
+    });
+
+    describe('exitGameToTxType', () => {
+        beforeEach(async () => {
+            this.dummyTxType = 1;
+            await this.registry.registerExitGame(this.dummyTxType, this.dummyExitGameAddress);
+        });
+
+        it('can receive tx type with exit game contract address', async () => {
+            expect(await this.registry.exitGameToTxType(this.dummyExitGameAddress))
+                .to.be.bignumber.equal(new BN(this.dummyTxType));
+        });
+    });
+
+    describe('registerExitGame', () => {
+        it('can register successfully', async () => {
```

Comment:
I understand that looking from bigger perspective this test and previous two are different behaviors but they look the same because of implementation details. I would go with just this test instead of the previous two. But I don't have a strong opinion on that.